docs/how-tos/howto-stake-unstake.md

```diff
@@ -0,0 +1,130 @@
+---
+id: howto-stake-unstake
+title: How to set up and use Proof of Stake (PoS)
+---
+
+## Overview
+
+This guide goes into detail on how to set up a Proof of Stake network with the Polygon SDK, how to stake funds for 
+nodes to become validators and how to unstake funds.
+
+It **highly encouraged** to read and go through the [setup IBFT locally](/docs//how-tos/howto-setup-ibft/howto-set-ibft-locally)
+/ [setup IBFT on the cloud](/docs//how-tos/howto-setup-ibft/howto-set-ibft-on-the-cloud) sections, before going along with this PoS guide. 
+These sections outline the steps needed to start a Proof of Authority (PoA) cluster with the Polygon SDK.
+
+:::warning PoA and PoS chains incompatible
+A chain that has previously been running in PoA mode, cannot be converted to a PoS chain during execution, and vice versa.
+
+The only way to switch IBFT mechanisms is to reset the chain and start it in the corresponding mode (PoA or PoS) from scratch.
+:::
+
+Currently, there is no limit to the number of validators that can stake funds on the Staking Smart Contract.
+
+## Staking Smart Contract
+
+The repo for the Staking Smart Contract is located [here](https://github.com/0xPolygon/staking-contracts).
+
+It holds the necessary testing scripts, ABI files and most importantly the Staking Smart Contract itself.
+
+## Setting up an N node cluster
+
+Setting up a cluster with the Polygon SDK is covered in the [setup IBFT locally](/docs/how-tos/howto-setup-ibft/howto-set-ibft-locally)
+/ [setup IBFT on the cloud](/docs/how-tos/howto-setup-ibft/howto-set-ibft-on-the-cloud) sections.
+
+The **only difference** between setting up a PoS and PoA cluster is in the genesis generation part.
+
+**When generating the genesis file for a PoS cluster, an additional flag is needed `--pos`**:
+```bash
+go run main.go genesis --pos ...
+```
+
+## Setting the length of an epoch
+
+Epochs are covered in detail in the [Epoch Blocks](/docs/guides/pos-concepts#epoch-blocks) section.
+
+To set the size of an epoch for a cluster (in blocks), when generating the genesis file, an additional flag is 
+specificed `--epoch-size`:
+```bash
+go run main.go genesis --epoch-size 50
+```
+This value specifies in the genesis file that the epoch size should be `50` blocks.
+
+The default value for the size of an epoch (in blocks) is `100000`.
```

Comment:
Add a comment that this is because of backwards compatibility, and that it is highly recommended to change this to a lower value so that validator set updates don't take 13+ hours to apply.

Reply:
Good call 💯 

I've highlighted this exact concept now 👍 

bdf6ba4